templates/categories.yaml

```diff
@@ -15,3 +15,4 @@ subCategoryMapping:
   backup: Backup
   dc: Direct Connect
   mlplatform: ML Platform
+  monitoring: Monitoring
```

Comment:
Should be named "Cloud Monitoring". We try to name services here accordingly their official cloud names (`https://cloud.vk.com/docs/monitoring-services/monitoring`).

Reply:
rename